mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -265,9 +158,9 @@ private static void checkResiduals(CombinedScanTask task) {
       Expression residual = fileScanTask.residual();
       if (residual != null && !residual.equals(Expressions.alwaysTrue())) {
         throw new UnsupportedOperationException(
-            String.format(
-                "Filter expression %s is not completely satisfied. Additional rows " +
-                    "can be returned not satisfied by the filter expression", residual));
+                String.format(
+                        "Filter expression %s is not completely satisfied. Additional rows " +
+                                "can be returned not satisfied by the filter expression", residual));
```

Comment:
I think the indentation was correct before. Can you revert this?

Reply:
Yes, sorry!